package ua.examples.practice.practice4;

import org.junit.Test;

public class Part5Test {

    @Test
    public void testMain() throws Exception {
    }

    @Test
    public void testRetrieveValue() throws Exception {
    }
}